stix_shifter_modules/cybereason/stix_transmission/connector.py

```diff
@@ -42,7 +42,7 @@ async def create_results_connection(self, query, offset, length):
                 query = json.dumps(query)
             response_wrapper = await self.api_client.get_search_results(query)
             if response_wrapper.response.history:                       # If the authentication is invalid, the history
-                if response_wrapper.response.history[0].status_code == 302:  # will be returned with 302 status code.
+                if response_wrapper.response.history[0].status == 302:  # will be returned with 302 status code.
```

Comment:
response_wrapper does not have response.history arguments anymore
you can take a look at any other connector ([example](https://github.com/opencybersecurityalliance/stix-shifter/blob/release/5.0.x/stix_shifter_modules/qradar/stix_transmission/results_connector.py#L13))
if this section is not covered with unit test please consider covering them

Reply:
response_wrapper contains response.history value, we tested it.
While doing valid authentication it contains empty list.
Invalid authentication it will contains the below value.
(<ClientResponse(https://integration.cybereason.net:8443/rest/visualsearch/query/simple) [302 ]><CIMultiDictProxy('Strict-Transport-Security': 'max-age=31536000;includeSubDomains', 'X-Frame-Options': 'DENY', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Cache-Control': 'no-cache, no-store, must-revalidate', 'Set-Cookie': 'JSESSIONID=5FFE7006108592557222F6706485539F; Max-Age=28800; Expires=Thu, 02-Feb-2023 15:17:37 GMT; Path=/; Secure; HttpOnly', 'Location': '/login.html', 'Content-Length': '0', 'Date': 'Thu, 02 Feb 2023 07:17:37 GMT')>,)

This field is required for identify the invalid authentication.